src/pages/EditUser.js

```diff
@@ -0,0 +1,349 @@
+import React, { useRef, useState } from 'react'
+import { Formik} from "formik"
+import * as yup from "yup"
+import { Form, Button, Container, Row, Col, Modal } from "react-bootstrap";
+import { FaUserCircle } from 'react-icons/fa'
+import { IconContext } from 'react-icons/lib'
+import AuthService from "../services/auth.service.js"
+import "../styles/pages/EditUser.css"
+import { useHistory } from 'react-router-dom';
+import { LinkContainer } from 'react-router-bootstrap'
+
+const schema = yup.object().shape({
```

Comment:
let's move this to a different folder and file so the component Is easier to read